### ISSUE TITLE
Oceanex decimal_to_precision AssertionError

### ISSUE THREAD

**Issue by erasmush** [ccxt/ccxt#8891]
- OS: Linux Ubuntu
- Programming Language version: Python 3.6
- CCXT version: 1.47.2
- Exchange: Oceanex
- Method: decimal_to_precision/create_limit_order

Hi, I keep getting an AssertionError whenever I try to place an order on Oceanex.

Any help would be greatly appreciated. 

```
import ccxt

Oceanex = ccxt.oceanex()

Oceanex.apiKey = ''
with open('/home//key.pem', "rb") as private_file:
    Oceanex.secret = private_file.read().decode()

Oceanex.load_markets()

print(Oceanex.markets['ADA/USDT']['precision'])
Oceanex.verbose = True
print(Oceanex.amount_to_precision(symbol='ADA/USDT', amount=50))
```

```
{'amount': '1', 'price': '5', 'base': '8', 'quote': '8'}
Traceback (most recent call last):
  File "ccxt_Test.py", line 13, in <module>
    print(Oceanex.amount_to_precision(symbol='ADA/USDT', amount=50))
  File "/usr/local/lib/python3.6/dist-packages/ccxt/base/exchange.py", line 1312, in amount_to_precision
    return self.decimal_to_precision(amount, TRUNCATE, self.markets[symbol]['precision']['amount'], self.precisionMode, self.paddingMode)
  File "/usr/local/lib/python3.6/dist-packages/ccxt/base/decimal_to_precision.py", line 41, in decimal_to_precision
    assert(isinstance(precision, numbers.Integral))
AssertionError

**Comment by kroitor** [comment]
@erasmush thank you for reporting, pushed a fix for this issue (the new build 1.47.6 will arrive in ~20 minutes). Please, let us know if you have this issue after upgrading to 1.47.6. Thx again!

**Comment by erasmush** [comment]
Hi Kroitor, thank you very much for resolving the issue.
